src/libstore/path.hh

```diff
@@ -61,6 +61,7 @@ struct StorePath : rust::Value<3 * sizeof(void *) + 24, ffi_StorePath_drop>
 
 typedef std::set<StorePath> StorePathSet;
 typedef std::vector<StorePath> StorePaths;
+typedef std::map<string, StorePath> StorePathMap;
```

Comment:
I might call this `OutputPathMap` instead, as `StorePathMap` sounds to me like `std::map<StorePath, ???>`.

Reply:
Indeed, that makes sense. Changed in 53edff29